mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/ui/AccountsFragment.kt

```diff
@@ -8,129 +8,42 @@ import android.view.View
 import android.view.ViewGroup
 import android.widget.ImageView
 import android.widget.TextView
-import androidx.recyclerview.widget.DividerItemDecoration
-import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.fragment.app.viewModels
+import androidx.hilt.navigation.compose.hiltViewModel
 import androidx.recyclerview.widget.RecyclerView
 import butterknife.BindView
 import butterknife.ButterKnife
-import butterknife.OnClick
 import dagger.hilt.android.AndroidEntryPoint
 import com.mifos.mobilewallet.model.domain.BankAccountDetails
 import org.mifos.mobilewallet.mifospay.R
+import org.mifos.mobilewallet.mifospay.bank.composeScreen.AccountScreen
 import org.mifos.mobilewallet.mifospay.bank.BankContract
-import org.mifos.mobilewallet.mifospay.bank.BankContract.BankAccountsView
 import org.mifos.mobilewallet.mifospay.bank.adapters.BankAccountsAdapter
 import org.mifos.mobilewallet.mifospay.bank.presenter.BankAccountsPresenter
+import org.mifos.mobilewallet.mifospay.bank.viewmodel.BankAccountsViewModel
 import org.mifos.mobilewallet.mifospay.base.BaseFragment
+import org.mifos.mobilewallet.mifospay.databinding.FragmentAccountsBinding
 import org.mifos.mobilewallet.mifospay.utils.Constants
 import org.mifos.mobilewallet.mifospay.utils.DebugUtil
-import org.mifos.mobilewallet.mifospay.utils.RecyclerItemClickListener
 import javax.inject.Inject
 
 @AndroidEntryPoint
-class AccountsFragment : BaseFragment(), BankAccountsView {
-    @JvmField
-    @BindView(R.id.inc_state_view)
-    var vStateView: View? = null
+class AccountsFragment : BaseFragment() {
 
-    @JvmField
-    @Inject
-    var mPresenter: BankAccountsPresenter? = null
-    var mBankAccountsPresenter: BankContract.BankAccountsPresenter? = null
-
-    @JvmField
-    @BindView(R.id.rv_accounts)
-    var mRvLinkedBankAccounts: RecyclerView? = null
-
-    @JvmField
-    @BindView(R.id.iv_empty_no_transaction_history)
-    var ivTransactionsStateIcon: ImageView? = null
-
-    @JvmField
-    @BindView(R.id.tv_empty_no_transaction_history_title)
-    var tvTransactionsStateTitle: TextView? = null
-
-    @JvmField
-    @Inject
-    var mBankAccountsAdapter: BankAccountsAdapter? = null
-
-    @JvmField
-    @BindView(R.id.linked_bank_account_text)
-    var linkedAccountsText: TextView? = null
-
-    @JvmField
-    @BindView(R.id.tv_empty_no_transaction_history_subtitle)
-    var tvTransactionsStateSubtitle: TextView? = null
+    lateinit var binding: FragmentAccountsBinding
```

Comment:
can we get rid of binding ? with compose we shouldn't need this

Reply:
it shall go while all the component of FinanceFragment is converted into jetpack compose.